src/Views/IframeView.php

```diff
@@ -271,8 +271,9 @@ private function getButtonHTML()
             '<div class="js-give-embed-form-modal-launcher-wrap">
 				<button
 				type="button"
-				class="js-give-embed-form-modal-opener"
-				data-form-id="%1$s"%3$s>%2$s</button>
+				class="js-give-embed-form-modal-opener givewp-donation-form-modal__open"
+				data-form-id="%1$s"%3$s
+				>%2$s</button>
```

Comment:
Let's add an `unreleased` tag in the top of this function to document this change.